market-generator/src/test/java/market/MarketSubscriptionManagerTest.java

```diff
@@ -0,0 +1,159 @@
+package market;
+
+import com.market.MarketDataRequest;
+import com.market.TickResponse;
+import com.market.banica.generator.exception.NoSuchGoodException;
+import com.market.banica.generator.service.MarketSubscriptionManager;
+import io.grpc.stub.StreamObserver;
+import org.junit.Assert;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.HashSet;
+
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
there are settings in intellij for not importing with *, it is not a good practice